src/openqaoa-core/openqaoa/utilities.py

```diff
@@ -1984,3 +1984,38 @@ def to_bin(number, n_qubits):
     ) / np.sqrt(len(wavefn_locs))
 
     return wavefunction
+
+def is_close_statevector(statevector1, statevector2) -> bool:
```

Comment:
We need to hint the type here, both should be numpy arrays.

Reply:
The issue is fixed by following commit:
https://github.com/tech-sketch/openqaoa/commit/c19528bb80fd717fe789a92080c71e920d5f26be